pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -929,7 +933,7 @@ cdef class _Timedelta(timedelta):
     def nanoseconds(self):
         """
         Return the number of nanoseconds (n), where 0 <= n < 1 microsecond.
-       
+
```

Comment:
Nice.  Want to add a line to lint.sh that checks for trailing white space in cython files?

Reply:
Sure!